src/components/editions/byline.tsx

```diff
@@ -91,6 +105,18 @@ const bylineSecondaryStyles = (large?: boolean): SerializedStyles => css`
 		: standardTextStyles('italic', 'light')}
 `;
 
+const getStyles = (format: Format, kickerColor: string): SerializedStyles => {
+	if (format.display === Display.Showcase) {
```

Comment:
Should we set out the Format order? Most Interviews have a Showcase display so Interview will need to be above Showcase.

Reply:
Good shout! Should be fixed now.